### ISSUE TITLE
Exception when extension doesn't have additional config

### ISSUE THREAD

**Issue by ildus** [funbringer/norsu#2]
When an extension doesn't have EXTRA_REGRESS_OPTS in Makefile, norsu will fail with TypeError exception. This is happenning because `run_temp` function will get [None] instead of empty list.

```
Traceback (most recent call last):
  File "/home/ildus/.local/bin/norsu", line 11, in <module>
    load_entry_point('norsu==0.4.5', 'console_scripts', 'norsu')()
  File "/home/ildus/.local/lib/python3.7/site-packages/norsu/main.py", line 282, in main
    command(parsed_args, extra)
  File "/home/ildus/.local/lib/python3.7/site-packages/norsu/main.py", line 193, in cmd_pgxs
    with run_temp(instance, config_files=config_files, port=port) as node:
  File "/usr/lib/python3.7/contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "/home/ildus/.local/lib/python3.7/site-packages/norsu/instance.py", line 409, in run_temp
    print('Found custom config:', os.path.basename(path), file=sys.stderr)
  File "/usr/lib/python3.7/posixpath.py", line 146, in basename
    p = os.fspath(p)
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

**Comment by ildus** [comment]
There are two place to be fixed: `cmd_pgxs` and `cmd_run` functions

**Comment by funbringer** [comment]
Fixed in 0.4.6 (already uploaded to PyPI).
